app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -41,7 +41,7 @@
 }
 
 @addressColumnHeaderLink = {
-    <a href="@agentAppointment.routes.AppointAgentController.sortRevokePropertiesByAddress(pagination, agentCode)" class="govuk-link" role="link" style="color: black">
+    <a href="@agentAppointment.routes.AppointAgentController.sortRevokePropertiesByAddress(pagination, agentCode)" role="link">
         @messages(s"propertyRepresentation.th.${ADDRESS.toString}")
```

Comment:
Do you have to camel case the address: @messages(s"propertyRepresentation.th.${ADDRESS.toString}")?

Reply:
good question

Reply:
https://github.com/hmrc/voa-property-linking-frontend/blob/main/conf/messages#L568 value of the ADDRESS key is camel case
By the way I did not tocuhed  any message key/values as nothing to do with scope of the story

Reply:
sorry, I misread that.